packages/react-mutation-mapper/src/util/FilterUtils.ts

```diff
@@ -152,8 +152,10 @@ export function applyDefaultMutationFilter(
     const filterPredicates = filter.values.map((value: MutationFilterValue) => {
         const valuePredicates = Object.keys(value).map(key =>
             includesSearchTextIgnoreCase(
-                mutation[key] ? mutation[key].toString() : undefined,
-                value[key] ? value[key.toString()] : undefined
+                (mutation as any)[key]
```

Comment:
It's not a `Mutation` type?

Reply:
It is, but `key` can be technically anything here. This was passing previously because `suppressImplicitAnyIndexErrors` was set to true for some reason. I have updated that in this PR to make the packages more type-safe.